compiler/tflchef/tflite/src/RecipeChef.cpp

```diff
@@ -74,254 +74,276 @@ std::unique_ptr<ModelRecipe> generate_recipe(const tflite::Model *model)
 
   TFliteImport tflite_import(model);
 
-  assert(tflite_import.num_subgraph() == 1);
-  tflite_import.select_sub_graph(0);
+  auto const num_subgraph = tflite_import.num_subgraph();
 
-  auto tensors = tflite_import.tensors();
-  auto buffers = tflite_import.buffers();
-  auto operators = tflite_import.operators();
-
-  // operand fillers for adding all operators
-  for (uint32_t i = 0; i < operators->size(); ++i)
-  {
-    const auto *op = operators->Get(i);
-    tflite::BuiltinOperator builtincode = tflite_import.builtin_code(op);
-
-    if (const auto *graph_builder = TFliteOpRegistry::get().lookup(builtincode))
-    {
-      graph_builder->filler(op, &tflite_import, model_recipe.get());
-    }
-    else
-    {
-      std::string opcodename = tflite_import.opcode_name(op);
-      throw std::runtime_error{"Not supported: " + opcodename};
-    }
-  }
-
-  // add all operands(tensors)
-  for (uint32_t i = 0; i < tensors->size(); ++i)
+  RecipeChefContext ctx;
+  for (uint32_t n = 0; n < num_subgraph; ++n)
   {
-    auto tensor = tensors->Get(i);
-
-    // check buffer
-    if (tensor->buffer() >= buffers->size())
-      throw std::runtime_error{"file load failed"};
+    tflite_import.select_sub_graph(n);
 
-    ::tflchef::Operand *operand = model_recipe->add_operand();
+    auto tensors = tflite_import.tensors();
+    auto buffers = tflite_import.buffers();
+    auto operators = tflite_import.operators();
 
-    operand->set_name(mio::tflite::tensor_name(tensor));
-    operand->set_type(as_tflchef_type(tensor->type()));
-    operand->set_is_variable(tensor->is_variable());
+    tflchef::Graph *graph;
```

Comment:
`graph` will be uninitialized when `n == 0`

Reply:
Done